server/src/main/java/io/druid/server/initialization/BatchDataSegmentAnnouncerConfig.java

```diff
@@ -37,6 +37,18 @@
   @Min(1024)
   private long maxBytesPerNode = 512 * 1024;
 
+  // Skip LoadSpec from segment announcements
+  @JsonProperty
+  private boolean skipLoadSpec = false;
+
+  // Skip dimension list from segment announcements
+  @JsonProperty
+  private boolean skipDimensions = false;
```

Comment:
added separate config in case someone has any tools built on top which needs metrics list and not dimensions list or vice versa.